File: angular/src/interfaces/calibration.ts
export interface ICalibration {
    id: number,
    repFirstName: string,
    repLastName: string,
    leaderUserId: number,
    groupScoreEarned: number,
    groupScorePossible: number,
    contactChannelId: number,
    calibrationDate: Date,
    contactId: string,
    isOpen: boolean,
    formId:number,
}

File: dotnet/CalibrationApp/CalibrationApp/Models/Calibration.cs
namespace CalibrationApp.Models
{
    public class Calibration
    {
        public int Id { get; set; }
        public string RepFirstName { get; set; }
        public string RepLastName { get; set; }
        public int FormId { get; set; }
        public decimal GroupScoreEarned { get; set; }
        public decimal GroupScorePossible { get; set; }
        public int ContactChannelId { get; set; }
        public DateTime CalibrationDate { get; set; }
        public string ContactId { get; set; }
        public bool IsOpen { get; set; }
        public int LeaderUserId { get; set; }
    }
}
